DATA_AND_PRIVACY.md

```diff
@@ -0,0 +1,988 @@
+# PowerToys Data & Privacy
+## Overview
+PowerToys diagnostic data is completely optional for users and is off by default. Our team believes in transparency and trust. As PowerToys is open source, all of our diagnostic data events are in the codebase.
+
+Additionally, this document aims to list each diagnostic data event individually and describe their purpose clearly.
+
+For more information, please read the [Microsoft privacy statement](https://privacy.microsoft.com/privacystatement). 
+
+## What does PowerToys collect?
+
+1. **Usage**: Understanding usage and frequency rates for utilities and settings helps us make decisions on where to focus our time and energy. This data also helps us better understand what and how to move well-loved features directly into Windows!
+2. **Stability**: Monitoring bugs and system crashes, as well as analyzing GitHub issue reports, assists us in prioritizing the most urgent issues.
+3. **Performance**: Assessing the performance of PowerToys features to load and execute gives us an understanding of what surfaces are causing slowdowns. This supports our commitment to providing you with tools that are both speedy and effective.
+
+### Success Story: Fixing FancyZones Bugs with Your Help
+FancyZones had numerous bug reports related to virtual desktop interactions. Initially, these were considered lower priority, since the assumption was that virtual desktops were not widely used, so we chose to focus on more urgent issues. However, the volume of bug reports suggested otherwise, prompting us to add additional diagnostics to see virtual desktop usage with FancyZones. We discovered that virtual desktop usage was much higher among FancyZones users. This new understanding led us to prioritize this class of bugs and get them fixed.
+ 
+## Transparency and Public Sharing
+As much as possible, we aim to share the results of diagnostic data publicly.
+
+We hope this document provides clarity on why and how we collect diagnostic data to improve PowerToys for our users. If you have any questions or concerns, please feel free to reach out to us.
+
+Thank you for using PowerToys!
+
+## List of Diagnostic Data Events
+_**Note:** We're in the process of updating this section with more events and their descriptions. We aim to keep this list current by adding any new diagnostic data events as they become available._
+
+_If you want to find diagnostic data events in the source code, these two links will be good starting points based on the source code's langauge._
+- [C# events](https://github.com/search?q=repo%3Amicrosoft/PowerToys%20EventBase&type=code)
+- [C++ events](https://github.com/search?q=repo%3Amicrosoft%2FPowerToys+ProjectTelemetryPrivacyDataTag&type=code)
+
+### General
+<table style="width:100%">
```

Comment:
in a future PR, these should be migrated to markdown i think.  having the hybrid html / markdown i don't think solves anything as github doesn't honor the 100%